openhabian-setup.sh

```diff
@@ -125,6 +125,7 @@ else
   load_create_config
   openhabian_console_check
   openhabian_update_check
+  fix_openhab_repo
```

Comment:
Do we want to add a check to see if the function has already been run so that it does not pop up every time even if it has already been performed?

Reply:
here you are